plugins/katello/3.16/user_guide/remote_databases/index.md

```diff
@@ -26,6 +26,15 @@ There are two ways to deploy Foreman with remote databases:
 
 In either scenario, both of the databases don't have to be remote. You can opt to use only a remote mongo database or only a remote postgresql database. Both postgresql and mongo databases can be on the same host, but this isn't recommended due to the amount of resources mongo can use.
 
+
+### Important notes:
+
+* Pulpcore installs a separate PostgreSQL database. Databases should be upgraded to avoid conflicts.
+
+* Katello 3.16 adds a dependency on [postgresql-evr](https://github.com/katello/postgresql-evr). The Katello repository provides the `rh-postgresql12-postgresql-evr` package on EL7. This should be installed on any external PostgreSQL v12 database before applying any db migrations.
+
+Note that [Foreman v2.1 requires PostgreSQL v10+](https://theforeman.org/manuals/2.1/index.html#Headlinefeatures).
```

Comment:
It was part of version 2.0, but it's not a headline feature in 2.1. The link should be removed but I'd also consider dropping it altogether. The evr package is only built for PG 12 so that already narrows it down for most users.

Reply:
The link is still incorrect.